cmd/loop/loopin.go

```diff
@@ -69,7 +69,7 @@ func loopIn(ctx *cli.Context) error {
 	}
 
 	limits := getInLimits(amt, quote)
-	err = displayLimits(loop.TypeIn, amt, limits, external)
+	err = displayLimits(looptypes.TypeIn, amt, limits, external)
```

Comment:
If possible we make sure that all commits build. Seeing this change, the previous commit probably doesn't.